### PR TITLE
Rebalance guitars (and similar instruments) as melee weapons.

### PR DESCRIPTION
**Pull request by Aerin-of-the-Toast**
#### Summary
Balance "Make the apocalypse a bit more metal by rebalancing several instruments as melee weapons"

#### Purpose of change

Guitars are heavy, break pretty easily when slammed into things, and are quite unwieldy as weapons, so it was bizarre that their stats before did a very paltry amount of bashing damage (less than a pipe or cudgel despite being way larger) and had +2 to hit.

#### Describe the solution

Jacked up the damages of guitars by quite a lot; electric guitar (26 bash) and acoustic guitar (18 bash) damages were suggested by mlangsdorf and damages for other guitar-ish weapons (violin, banjo, ukulele) were also increased. To-hit nerfed across the board from +2 to -1 in accordance with the game balance doc. All were given FRAGILE_MELEE because if you've ever watched a video of a concert where they smash guitars on the ground, they only last a couple of swings.

EDIT: Fixed the to-hit of the golden fiddle as well, since I forgot to do that the first time around, and made sure the to-hit followed the new formatting. Didn't adjust the damage of the golden fiddle though, I'm not entirely sure how much damage a solid-gold violin would even do, if you could even swing it...

#### Describe alternatives you've considered

Embracing the spirit of heavy metal and giving them outright battle axe stats. This is a joke. I'm also half tempted to make the acoustic and electric guitars valid Fior Di Battaglia weapons, but that might be too absurd.

#### Testing

Tested with debug, stats were adjusted correctly.

#### Additional context

DPS still is pitifully low for all of them due to the very slow speed and bad accuracy, all below the DPS of a pipe, but at least now when you channel your inner Jack Black and start hitting zombies with them they'll at least do good damage when you *do* hit.

### CODE REVIEW

File: data/json/items/tool/musical_instruments.json [hunk 1]
```diff
@@ -10,8 +10,9 @@
     "longest_side": "72 cm",
     "price": 7500,
     "price_postapoc": 500,
-    "to_hit": 2,
-    "bashing": 7,
+    "to_hit": -1,
```
Comment (anothersimulacrum) [inline data/json/items/tool/musical_instruments.json:13]:
```suggestion
    "to_hit": { "grip": "none", "length": "short", "surface": "any", "balance": "clumsy" },   
```

File: data/json/items/tool/musical_instruments.json [hunk 2]
```diff
@@ -159,8 +160,9 @@
     "longest_side": "58 cm",
     "price": 7500,
     "price_postapoc": 250,
-    "to_hit": 2,
-    "bashing": 5,
+    "to_hit": -1,
```
Comment (anothersimulacrum) [inline data/json/items/tool/musical_instruments.json:163]:
```suggestion
    "to_hit": { "grip": "none", "length": "short", "surface": "any", "balance": "clumsy" },   
```

File: data/json/items/tool/musical_instruments.json [hunk 3]
```diff
@@ -189,8 +191,9 @@
     "longest_side": "60 cm",
     "price": 7500,
     "price_postapoc": 600,
-    "to_hit": 2,
-    "bashing": 5,
+    "to_hit": -1,
```
Comment (anothersimulacrum) [inline data/json/items/tool/musical_instruments.json:194]:
```suggestion
    "to_hit": { "grip": "none", "length": "short", "surface": "any", "balance": "clumsy" },   
```

File: data/json/items/tool/musical_instruments.json [hunk 4]
```diff
@@ -273,7 +276,7 @@
     "category": "tools",
     "weight": "2200 g",
     "color": "brown",
-    "to_hit": 2,
+    "to_hit": -1,
```
Comment (anothersimulacrum) [inline data/json/items/tool/musical_instruments.json:279]:
```suggestion
    "to_hit": { "grip": "none", "length": "short", "surface": "any", "balance": "clumsy" },   
```

File: data/json/items/tool/musical_instruments.json [hunk 5]
```diff
@@ -308,7 +311,7 @@
     "category": "tools",
     "weight": "3750 g",
     "color": "red",
-    "to_hit": 2,
+    "to_hit": -1,
```
Comment (anothersimulacrum) [inline data/json/items/tool/musical_instruments.json:314]:
```suggestion
    "to_hit": { "grip": "none", "length": "short", "surface": "any", "balance": "clumsy" },   
```

**Review by anothersimulacrum — COMMENTED** [review]
Please update the to-hit to use the new format. I chose values based on what I thought was correct, they may not be what you would put.
I get different values than -1 (-2), probably because of a different values.

File: data/json/items/tool/musical_instruments.json [hunk 6]
```diff
@@ -308,7 +311,7 @@
     "category": "tools",
     "weight": "3750 g",
     "color": "red",
-    "to_hit": 2,
+    "to_hit": { "grip": "none", "length": "short", "surface": "any", "balance": "clumsy" },
```
Comment (anothersimulacrum) [inline data/json/items/tool/musical_instruments.json:314]: I think you committed the wrong one.
```suggestion
    "to_hit": { "grip": "none", "length": "long", "surface": "any", "balance": "clumsy" },
```

**Comment by pehamm** [comment]
Shouldn't balance be `uneven` rather than `clumsy`? I just grabbed two of my guitars and the weight distribution absolutely allows for swinging.

**Pull request merged**